### DISCUSSION TITLE
Question about camera quanity limit

### DISCUSSION THREAD

**Discussion by zekedz123** [jyjblrd/Mocap-Drones#26] [Q&A]
Hi i would like to ask if the camera is only limited to 4? Or is it possible to for example use 6/8/12/16 cameras for larger sized room/ setting? Thank you 🙏

**Comment by jyjblrd** [comment]
Hi, the system has no limit on the number of cameras it can use. However, some of my algorithms (such as the point correlation code) may not scale very well to 8/12/14 cameras, and may need to be rewritten.
